wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -165,12 +165,33 @@ const SingleRoute = (props: Props) => {
     const routeConfig = RouteOperator.getRoute(props.config.route);
 
     return !routeConfig.AUTOMATIC_DEPOSIT;
-  }, [props.config.route, destChain]);
+  }, [props.config.route]);
+
+  const errorMessage = useMemo(() => {
+    if (!props.error) {
+      return null;
+    }
+
+    return (
+      <>
+        <Divider flexItem sx={{ marginTop: '8px' }} />
+        <Stack direction="row" alignItems="center">
+          <WarningIcon htmlColor={theme.palette.error.main} />
+          <Stack sx={{ padding: '16px' }}>
+            <Typography color={theme.palette.error.main} fontSize={14}>
+              {props.error}
```

Comment:
I don't think exposing the raw `error.message` from `isRouteAvailable` in the UI is a good idea. there's no guarantee it's human-friendly. it could be a crazy looking network error. we should have some code that tries parsing it into a human-friendly string, and defaults to something generic like "Route is unavailable, try again later."

Reply:
@artursapek the ideal solution seems to be SDK adding error codes (or anything) to distinguies human readable ones from others. Until then I have two alternatives to detect whether an error message is non-html:

Using the DOMParser:
```
var errorRepsonse = new DOMParser().parseFromString(
  e.message,
  'text/html',
);
const isHtml = Array.from(errorRepsonse.body.childNodes).some(
  (node) => node.nodeType === 1,
);
```

Or just simple RegEx:
```
const hasHTMLTags = /<\/?[a-z][\s\S]*>/i.test(e.message);
```

WDYT?

Reply:
(discussed privately - SDK needs to normalize errors to a standard set. until then, use generic message to avoid leaking technical errors that confuse users)